components/net/filemanager_thread.rs

```diff
@@ -163,8 +163,10 @@ impl FileManager {
             FileManagerThreadMsg::DecRef(id, origin, sender) => {
                 let _ = sender.send(self.store.dec_ref(&id, &origin));
             },
-            FileManagerThreadMsg::RevokeBlobURL(id, origin, sender) => {
-                let _ = sender.send(self.store.set_blob_url_validity(false, &id, &origin));
+            FileManagerThreadMsg::RevokeBlobURL(url, sender) => {
+                if let Ok((id, origin)) = parse_blob_url(&url) {
```

Comment:
Oh - we still are throwing away the fragment and query string from the URL, which is why the tests continue to fail. We need to explicitly deal with those.

Reply:
Thanks for the review.
I will check and fix it!